src/lib/utils.js

```diff
@@ -17,10 +17,13 @@ const modes = ['view', 'edit', 'create'];
 
 const validateMode = (mode) => modes.findIndex((m) => mode === m) > -1;
 
-const validateAndInvokeCallback = (callback, data) => {
+const validateAndInvokeCallback = (callback, data, eventDetail) => {
```

Comment:
Do you need an extra param ?. `report` is already available during onload right ?